src/pages/ServiceGraph/ServiceGraphPage.tsx

```diff
@@ -91,32 +61,21 @@ export default class ServiceGraphPage extends React.Component<RouteComponentProp
       );
     }
     return (
-      <PfContainerNavVertical>
-        <PfHeader>
-          <h2>
-            Services Graph for namespace:
-            <DropdownButton
-              id="namespace-selector"
-              title={this.state.graphNamespace}
-              onSelect={this.namespaceSelected}
-              style={{ marginLeft: 20 }}
-              bsSize="large"
-            >
-              {this.state.availableNamespaces.map(ns => (
-                <MenuItem key={ns.name} active={ns.name === this.state.graphNamespace} eventKey={ns.name}>
-                  {ns.name}
-                </MenuItem>
-              ))}
-            </DropdownButton>
-          </h2>
-          {alertsDiv}
-        </PfHeader>
+      <div className="container-fluid container-pf-nav-pf-vertical">
```

Comment:
Please use `PfContainerNavVertical` instead of `<div className="container-fluid container-pf-nav-pf-vertical">`. Trying to componentize as much of this as possible. If need to change a style we can just change it in the component and it will be reflected everywhere in the app. Also add `PfHeader`.